index.bs

```diff
@@ -0,0 +1,331 @@
+      media element's {{HTMLMediaElement/autoplayPolicy}} will be {{disallowed}}.
+
+      However, after a user clicks on a media element, then this media element's
+      {{HTMLMediaElement/autoplayPolicy}} might become {{allowed}} if a
+      user-agent decides to bless this element because that seems intended by
+      users. In this case, the document's {{Document/autoplayPolicy}} and other
+      media elements' {{HTMLMediaElement/autoplayPolicy}} (if any), which
+      haven't been clicked yet and are on the same document, will still be
+      {{disallowed}}.
+    </div>
+
+    <div class="note">
+      Note: If the document has child documents, then the result from child
+      documents could be different from the result of their parent documents
+      depending on the implementation.
+    </div>
+
+    <div class=example>
+      Assume that the top level document A in `foo.com` returns {{allowed}} and
+      it has an embedded iframe, which has another document B from `bar.com`. A
+      user-agent could either make child document B return the same result that
+      is inherited from the top level document A. Or make the document B return
+      a different result, eg. {{disallowed}}.
+
+      Doing the former helps to lower the complexity and make the behavior of
+      blocking autoplay more consistent. The latter helps providing a
+      finer-grained autoplay control.
+    </div>
+
+    <div class="note">
+      Note : In addition, document's {{Document/autoplayPolicy}} can also be
+      used to know whether [[webaudio inline]] can be allowed to start if a
+      user-agent blocks web audio by default.
+    </div>
+
+    <div class=example>
+      [Web audio](https://webaudio.github.io/web-audio-api/#allowed-to-start)
+      uses the [=sticky activation=] to determine if {{AudioContext}} can be
+      allowed to start. If the document is allowed to autoplay, then
+      {{AudioContext}} should also be allowed to start.
+
+      <pre class="lang-javascript">
+        var ac = new AudioContext;
+        if (document.autoplayPolicy == "allowed") {
+          ac.onstatechange = function() {
+            if (ac.state == "running") {
+              // Start running audio app
+            } else {
+              // Display a bit of UI to ask the user to start the audio app.
+              // Audio starts via calling ac.resume() from a handler, and
+              // 'onstatechange' allows knowing when the audio stack is ready.
+            }
+          }
+        }
+      </pre>
+    </div>
+
+  <h3 id="media-element-api">The HTMLMediaElement API</h3>
+    <pre class="idl">
+    partial interface HTMLMediaElement {
+      readonly attribute AutoplayPolicy autoplayPolicy;
+    };
+    </pre>
+    This represents the current status of whether this {{HTMLMediaElement}} is
+    allowed to autoplay or not.
+
+    <dl class="switch">
+      <dt>If the value is {{allowed}}</dt>
+      <dd>
+        This media element is allowed to autoplay.
```

Comment:
Ditto. Anything to be said about user gestures? I.e., "... allowed to autoplay within the current execution context."

Reply:
I wonder if we should add "...within the current execution context" for `allowed-muted`, not `allowed`? Because that somehow indicates that `allowed-muted` is a temporarily state.